src/frontend/app.ts

```diff
@@ -199,6 +199,7 @@ export default (octokitApp: App): Express => {
 	app.get(
 		"/github/setup",
 		csrfProtection,
+		oauth.checkGithubAuth,
```

Comment:
I wonder why it was missing and what was the effect?

Reply:
It wasn't 'missing' before, it just wasn't needed. Now that we are connecting the org from this page, we need to pass this middleware.